libopenage/renderer/stages/world/render_stage.cpp

```diff
@@ -58,6 +58,10 @@ void WorldRenderStage::update() {
 	std::unique_lock lock{this->mutex};
 	auto current_time = this->clock->get_real_time();
 	for (auto &obj : this->render_objects) {
+		if (!obj->within_camera_frustum(this->camera)) {
+			continue;
```

Comment:
This will still render the object btw, it just skips the update :D

I'll have to fix this in a separate PR as it is not possile right now to remove specific objects from the render pass.